docs/source/markdown/podman.1.md

```diff
@@ -75,110 +72,159 @@ If no identity file is provided and no user is given, podman defaults to the use
 Podman prompts for the login password on the remote server.
 
 Identity value resolution precedence:
- - command line value
- - environment variable `CONTAINER_SSHKEY`, if `CONTAINER_HOST` is found
- - `containers.conf`
-Remote connections use local containers.conf for default.
+ - command line value.
+ - environment variable `CONTAINER_SSHKEY`, if `CONTAINER_HOST` is found.
+ - `containers.conf`.
+Remote connections use local `containers.conf` for default.
+
+#### **--log-level**=***_error_** | debug | info | warn | fatal/panic*
 
-#### **--log-level**=*level*
+Log messages above a specified level.
 
-Log messages above specified level: debug, info, warn, error (default), fatal or panic (default: "error")
+| Argument    | Description |
+| ----------- | ----------- |
+| Error (Default) | Runtime/Unexpected error messages.|
```

Comment:
Error should just be **bold**, without the DEFAULT behind it.

Reply:
Fixed, please take another look for other tables as well

Reply:
Warn is now default.

Reply:
Updated